### PR TITLE
React Part 1

### PR DESCRIPTION
**Pull request by gildedgardenia**

### CODE REVIEW

File: content/react-lsn1/exercises/_index.md [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+The following exercises walk you through the process of modifying existing
+components and creating new ones.
+
+## Starter Code
+
+For this set of exercises, you will need a copy of the LaunchCode Angular
```
Comment (codinglikeagirl42) [inline content/react-lsn1/exercises/_index.md:20]:
```suggestion
For this set of exercises, you will need a copy of the LaunchCode React
```

File: content/react-lsn1/exercises/_index.md [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+```console
+   $ pwd
+      react-projects
+   $ ls
+      part1 part2 part3
+   $ cd part1
+   $ cd exercises
+```
+
+You should be able to run  `npm start`, but the page will not look pretty. Time to fix that!
```
Comment (codinglikeagirl42) [inline content/react-lsn1/exercises/_index.md:38]: Add a note about using npm install, to install the dependencies.

File: content/react-lsn1/exercises/_index.md [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+   font-size: 1.3vw;
+   text-align: left;
+}
+
+.movieHeading {
+   text-align: left;
+}
+```
+
+{{% /expand %}}
```
Comment (codinglikeagirl42) [inline content/react-lsn1/exercises/_index.md:67]:
```suggestion
  {{% expand "Check Your Solution" %}}
  
  ```css
  .movieText {
     color: purple;
     font-size: 1.3vw;
     text-align: left;
  }
  
  .movieHeading {
     text-align: left;
  }
  ```
  
  {{% /expand %}}
```

**Review by codinglikeagirl42 — APPROVED** [review]
Looks Good, just a few small suggestions to look at before merging.

File: content/react-lsn1/exercises/_index.md (hunk 2 above)
Comment (gildedgardenia) [reply content/react-lsn1/exercises/_index.md:38]: Done! Thank you!

File: content/react-lsn1/exercises/_index.md (hunk 3 above)
Comment (gildedgardenia) [reply content/react-lsn1/exercises/_index.md]: Done! Thanks!

**Pull request merged**